docs/content/direct/user-guide-intro.md

```diff
@@ -5,7 +5,22 @@ See the KubeStellar [overview](../readme.md) for architecture and other informat
 
 This user guide is an ongoing project. If you find errors, please point them out in our [Slack channel](https://kubernetes.slack.com/archives/C058SUSL5AA/) or open an issue in our [github repository](https://github.com/kubestellar/kubestellar)!
 
-Installing and using KubeStellar progresses through the following stages.
+## Quickstart
+
+If you want to try a simplified installation process and example, you can try our [Quickstart](get-started.md) using Kind and [a helm chart](core-chart.md).
```

Comment:
This is the sentence with the link to the helm chart that I would move to a subordinate sentence.
(apologies for not quite grokking the review interface before)

Reply:
Done